### DISCUSSION TITLE
How are you going to deal with prices in Queensland?

### DISCUSSION THREAD

**Discussion by jaymis** [amberelectric/public-api#113] [General]
I know this probably isn't techically the suggested channel for pricing discussions, but Amber seems like an open company, so I thought it'd be worth seeing if we could discuss this publicly.

I've only been with Amber for a couple of weeks, so maybe my experience is atypical, but I've barely seen prices near my old retailer pricing. It seems to be spending most of its time around double that, with plenty of time quite a lot higher, and I'm seeing huge spikes ($5-15 kw/h) every couple of days. Sometimes days in a row.

Is this an abberation that's going to settle back down, or is there something weird in Queensland's pricing? Are we like Texas (I think it was) in the USA?

**Comment by madpilot** [comment]
Unfortunately, you've signed up at a really bad time - a coal power station in Victoria went out of action about a month ago, and is causing pricing chaos - it will probably continue until August. In all honestly, you may be better off churning to another retailer until the prices settle down. Note though, that there is a price rise coming across the grid in July, and we expect all retailers to increase their prices significantly. For most retailers, the price rise happens once a year - for Amber, the price follows the market, so when it settles again, we should come out significantly cheaper again.

Thankfully, what happened in Texas can't happen in Australia - AEMO caps the number of days that a price spike can happen, and will then clamp the price to 40c if they keep happening. We also offer a DMO guarantee, so (over 12 months) you'll never pay more than the default market offer.

**Reply by jaymis** [reply]
You're working late @madpilot ;)

Thanks for the explanation. Guess I'll have a look around and see if I can go somewhere else temporarily. 

There's no publicly accessible API endpoint for pricing is there? It'd be nice to keep Amber in my Home Assistant setup so I could monitor for when prices come back down.

/edit: Just found [this discussion](https://github.com/amberelectric/public-api/discussions/104#discussioncomment-2305750) mentioning a partner API.

**Reply by madpilot** [reply]
Technically we don't shut off the API (or the App) when you churn, but we are filtering closed sites.

I'll fix that up, then you can just keep using it.

**Reply by jaymis** [reply]
Fortuitous timing, I just got an email newsletter yesterday from Chris the Amber co-founder talking about how you're going to deal with  the volatility by making early/monthly payouts for people who are over the DMO.

So I think instead of going elsewhere, I'll devote the effort I would have put into switching to doing some research on solar installations and systems that'll integrate well with Home Assistant.

**Comment by oldwindmill** [comment]
looks like the 'unprecedented times' are really stretching the aemo algorithms. all night the prices have been showing north of $15, but have resolved to (relatively) cheap 50c or so after the 30 mins. Makes for a interesting night though, wondering if cooking dinner is going to cost more than going out for it.
But seriously, the way i read the aemo dashboard suggests prices are basically pegged at $300/MWH, but the forcecasts are still showing $15000 for the next period. which i guess is where the app forecasts are coming from, but clearly ( i hope....) wildly wrong. 

is it likely to get better....or should i just close the app and get on with it?

russ

**Reply by madpilot** [reply]
The forecasts should now be showing the capped price of 30c/kWh for spot pricing.

We had to manually intervene yesterday, as we didn't realise the forecasts coming of the AEMO API would be the uncapped price, so there was a bit of a lag in the prices settling.

AEMO is still running forecasts as uncapped price - they are watching for the rolling uncapped price to fall back under the threshold before removing the cap. Though, looking at the market at the moment, that won't be happening for a while.

**Reply by oldwindmill** [reply]
mine have the singapore (ausnet) tax so showing 45c .i did notice earlier that the forecast for lunchtime seemed to reflect peak solar, dropping to 0.15 but that has disappeared again.

**Reply by madpilot** [reply]
The price can dip under the cap. Whether or not that price will come to fruition, is anyone's guess.
